src/main/java/db/Storage.java

```diff
@@ -0,0 +1,21 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class Storage {
+    private final List<FruitTransaction> transactions = new ArrayList<>();
```

Comment:
Inside storage it's better to have structure that looks like storage.
You anyway goes to Map by collecting results in FruitService. Would be nicer to stick to it from the start
```suggestion
    private final Map<String, Integer> fruitQuantityByName = new HashMap<>();
```